services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -159,13 +159,18 @@ private Map<String, String> supportedLocales(Properties messages) {
     
     private String messagesToJsonString(Properties props) {
         if (props == null) return "";
-        
-        JsonObjectBuilder json = Json.createObjectBuilder();
+        ObjectMapper mapper = new ObjectMapper();        
```

Comment:
Should be possible to replace with:

JsonSerialization.writeValueAsString(props)
```suggestion
        ObjectMapper mapper = new ObjectMapper();        
```
Using [JsonSerialization](https://github.com/keycloak/keycloak/blob/8ea8aaebcf3ec7166607f79141f40ba7a2eecfaa/core/src/main/java/org/keycloak/util/JsonSerialization.java#L38) uses a shared ObjectMapper + makes it possible to configure globally for the server if pretty-printing is used or not.

Reply:
@ebremer This is looking good. I tried creating a PR and chose the wrong json javax lib to remove. I think the only outstanding thing here is probably using JsonSerialization instead of ObjectMapper directly.